src/PageContent/RequestPickup/PickupSummary.js

```diff
@@ -47,16 +52,27 @@ class PickupSummary extends React.Component {
         let endTime = moment(this.props.request.endTimestamp).format(
             StringFormat.TIME
         );
+        let style;
+        if (this.props.type === DeliveryType.RECURRING) {
+            style = 'top-line-recurring';
+        } else {
+            style = 'top-line-emergency';
+        }
         return (
             <div className="backdrop">
                 {/* TODO: fix background opacity. Maybe with iFrame. */}
                 <div className="modal">
-                    <div className="top-line" />
+                    <div className={style} />
                     <p id="exit" onClick={this.props.onClose}>
                         &times;
                     </p>
                     <div className="summary-title flex">
-                        <img src={truck} alt="truck" />
+                        {style === 'top-line-recurring' ? (
+                            <img src={greenTruck} alt="green-truck" />
+                        ) : (
+                            <img src={redTruck} alt="red-truck" />
+                        )}
```

Comment:
like `style`, add `truckSrc` and `truckAlt` into the if statement. And just do
`<img src={truckSrc} alt={truckAlt} />`